src/main/app/idam-api/IdamAPI.ts

```diff
@@ -151,4 +151,24 @@ export class IdamAPI {
         return Promise.reject(errorMessage);
       });
   }
+
+  public getUsersWithRoles(roles: string[]): Promise<User[]> {
+    let queryString = '';
+    roles.forEach((role, index, roles) => {
+      queryString += 'roles:' + role;
+      if (index !== roles.length - 1) {
+        queryString += ' OR ';
+      }
+    });
+
+    return this.userAxios
+      .get(`/api/v1/users?size=500&query=(${queryString}) AND lastModified:>2018-01-01T00:00:00.000000`)
```

Comment:
I am sure there is reason for adding this lastModified string when getting users with roles, why do we need that?

Reply:
@KianKwa the query on the API side specifically requires a lastModified date to be specified otherwise it'll return an empty array.